src/components/User/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { createContext, useContext, useState } from 'react'
+import firebase from 'firebase'
+import { authentication } from '../../firebase'
+
+type UserContext = {
+	user: firebase.User | null
+	isLoggedIn: boolean
+}
+
+const User = createContext<UserContext>({
```

Comment:
I wouldn't name it `User`, because it simply represents a user context. :sweat_smile:
```suggestion
const UserContext = createContext<UserContext>({
```

Reply:
Ok 😅